docs/reset-password.md

```diff
@@ -0,0 +1,54 @@
+# Reset TechPass password
+
+If you are a vendor and if your TechPass domain is ```techpass.gov.sg```, you may reset your TechPass password according to the [Azure Password Policy](https://learn.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-policy#administrator-password-policy-differences) or if your TechPass account gets locked.
```

Comment:
I feel "or if your TechPass account gets locked" a bit out of place.

Reply:
@JLeeChan , May I check with you if it is out of place from a writing perspective?

Reply:
about the sentence structure.
info-wise it is useful

Reply:
ok, i have removed the latter part.